cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProfileHttpHandler.java

```diff
@@ -165,7 +169,7 @@ public void getProfile(HttpRequest request, HttpResponder responder,
                          @PathParam("namespace-id") String namespaceId,
                          @PathParam("profile-name") String profileName) throws Exception {
     ProfileId profileId = getValidatedProfile(namespaceId, profileName);
-    AuthorizationUtil.ensureAccess(profileId, authorizationEnforcer, authenticationContext.getPrincipal());
+    accessEnforcer.isVisible(profileId, authenticationContext.getPrincipal());
```

Comment:
If isVisible is a direct mapping to StandardPermissions.GET in all cases, it might be better to use enforce(StandardPermissions.GET) to make future deprecation work easier.

Reply:
I removed isVisible for single entity

Reply:
SGTM, thanks!